ci-operator/config/openshift/kube-state-metrics/master.json

```diff
@@ -0,0 +1,38 @@
+{
+  "tag_specification": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "origin-v3.11",
+    "tag": "",
+    "tag_overrides": {}
+  },
+  "base_images": {
+    "base": {
+      "cluster": "https://api.ci.openshift.org",
+      "namespace": "openshift",
+      "name": "origin-v3.11",
+      "tag": "base"
+    }
+  },
+  "test_base_image": {
+    "cluster": "https://api.ci.openshift.org",
+    "namespace": "openshift",
+    "name": "release",
+    "tag": "golang-1.10"
+  },
+  "canonical_go_repository": "k8s.io/kube-state-metrics",
+
+  "images": [
+    {
+      "from": "base",
+      "to": "kube-state-metrics"
```

Comment:
Ah, this seems to me, that it is using the `Dockerfile` in `master` instead of `openshift-master`. `openshift-master` is set in ` cluster/ci/config/prow/config.yaml`. Do I need to specify the branch in the `master.json` file as well, @smarterclayton?

Reply:
Oh, I ran against  the wrong branch.  Sorry.

Reply:
Having different release branches in our forks is really confusing... I think this is going to cause pain down the road, so we should really really really try to have "master" in our fork mean "the thing Openshift cares about"